src/index.js

```diff
@@ -1,28 +1,21 @@
 import './css/style.css';
+/* eslint-disable-next-line import/no-cycle */
+import validateForm from './function.js';
 import Icon from './images/option.png';
+import removeTask from './remove.js';
 
-const tasks = [
-  {
-    description: 'Create a new repository',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Create a local webpack project',
-    completed: true,
-    index: 1,
-  },
-  {
-    description: 'Checkout the linters',
-    completed: true,
-    index: 2,
-  },
-  {
-    description: 'Push your project on Github',
-    completed: false,
-    index: 3,
-  },
-];
+let tasks = [];
+
+export const main = document.querySelector('.container');
+const dataLoading = () => {
+  tasks = JSON.parse(localStorage.getItem('datas')) ?? [];
+  main.innerHTML = '';
+  /* eslint-disable */
+  document.body.appendChild(component());
+  /* eslint-enable */
```

Comment:
- [ ] Just as I pointed out above, I'll suggest you remove these linters disable rules and try to re-write the logic of the code to fix the issue. :wave: :+1: